coral-dev-board.Dockerfile

```diff
@@ -65,22 +68,15 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         python3-scipy \
         python3-wget \
     && rm -rf /var/lib/apt/lists/* \
-    && python3 -m pip install --upgrade pip setuptools==41.0.0 && pip install \
+    && python3 -m pip install --upgrade pip setuptools==41.0.0 && pip install -r /requirements.txt \
```

Comment:
I had an error on building this line on pandas installation
![Selection_015](https://user-images.githubusercontent.com/64582507/93902590-97669380-fd04-11ea-8376-22b4038904a3.png)

Reply:
Haven't seen that. I suppose it only happened on Coral? Did you happen to retry it just to check that it is not a temporal issue?
I can't test this on a Coral at the moment

Reply:
yeah, I retried but the same happened.

Reply:
this seems to be a memory issue and is solved using increasing swap size. so the docker and installation don't have problem.